dht/murmur3_partitioner.cc

```diff
@@ -110,9 +96,7 @@ inline int64_t long_token(token_view t) {
         return std::numeric_limits<long>::min();
```

Comment:
This `long` here makes me uncomfortable

Reply:
Correct, it should be int64_t. Can come in a separate patch.

Reply:
done.